cmd/podman/main.go

```diff
@@ -124,6 +124,11 @@ func main() {
 			Name:  "cpu-profile",
 			Usage: "path for the cpu profiling results",
 		},
+		cli.StringFlag{
+			Name:   "default-mounts-file",
+			Usage:  "path to default mounts file",
+			Hidden: true,
```

Comment:
just to be sure, did you do 'libpod --help' and not see this listed?

Reply:
yup, does not show up.